libkbfs/block_journal.go

```diff
@@ -574,6 +607,8 @@ func (j *blockJournal) removeReferences(
 		if count == 0 {
 			delete(j.refs, id)
 			if removeUnreferencedBlocks {
+				// TODO: Decrement j.blockBytes, but only if the block
```

Comment:
Any ideas on how to deal with this?  I figure it's not important if `blockBytes` is accurate for `BlockServerDisk`, but you might have a different opinion.